server/models/budgetModel.js

```diff
@@ -0,0 +1,30 @@
+const { DataTypes } = require("sequelize");
+const sequelize = require("../database/sequelize");
+
+const Budget = sequelize.define(
+  "budget",
+  {
+    id: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      primaryKey: true,
+    },
+    month: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+    year: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+    estimated_budget: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+  },
+  {
+    tableName: "budgets",
```

Comment:
```
  tableName: 'budget',
      underscored: true
```

Reply:
added in commit 9a6b72d22fc7b1046356ec2c70ed77825b0bb0b4